Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceUpdateExecutorInterface.h

```diff
@@ -22,6 +22,9 @@ namespace AzToolsFramework
             AZ_RTTI(InstanceUpdateExecutorInterface, "{26BB4ACF-6EA5-4668-84E7-AB75B90BA449}");
             virtual ~InstanceUpdateExecutorInterface() = default;
 
+            // Add an instance into the queue so it's updated at the end of the frame.
```

Comment:
nit: I'd change this comment to match the following one

```
// Add an instance into the queue so it's updated later.
```

Reply:
It can be resolved once #11120 is merged.